### ISSUE TITLE
Breaking change in ClosedXML v0.96 

### ISSUE THREAD

**Issue by igitur** [ClosedXML/ClosedXML.Report#194]
Hi,

Refer to https://github.com/ClosedXML/ClosedXML/pull/1396

This PR has been merged and is quite a big breaking change.

**Comment by b0bi79** [comment]
Thanks. I will try to update to v0.96 as soon as possible.

**Comment by igitur** [comment]
Thanks. 0.96 won't be released soon and I will give warning. But you can test against the `develop` branch in the meantime.

**Comment by sopelt** [comment]
Hi, this issue now seems to happen in the wild. Combining [ClosedXML.Report 0.2.4](https://www.nuget.org/packages/ClosedXML.Report/0.2.4) with [ClosedXML 0.96.0](https://www.nuget.org/packages/ClosedXML/0.96.0) results in 

```
The type 'IXLWorkbook' is defined in an assembly that is not referenced. You must add a reference to assembly 'ClosedXML, Version=0.95.0.0, Culture=neutral, PublicKeyToken=null'.
```

Any plans/progress for an update?

Thank you!

**Comment by d00lar** [comment]
any news?

**Comment by CumpsD** [comment]
When will ClosedXML.Report support ClosedXML 0.96?

**Comment by Pankraty** [comment]
There are two version released today: 0.2.6 supporting ClosedXML 0.97, and 0.2.5 (based on ClosedXML 0.96) as a fallback in case the latest version brings too many breaking changes.
